README.md

```diff
@@ -31,10 +31,12 @@ Once the dependencies are installed, you can build AIKIDO using [CMake]:
 ```shell
 $ mkdir build
 $ cd build
+$ source /opt/ros/indigo/setup.bash
```

Comment:
Yes, also we can move this comment to the very top: there's no reason to create the build directory first, and that way the remaining ROS-agnostic instructions are together.